src/status_im2/contexts/onboarding/common/intro/style.cljs

```diff
@@ -0,0 +1,60 @@
+(ns status-im2.contexts.onboarding.common.intro.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def progress-bar-container
+  {:margin-vertical 16
+   :flex-direction  :row})
+
+(defn get-color
+  [index position]
+  (if (= index position) colors/white colors/white-opa-10))
+
+(defn progress-bar-item
+  [index position end?]
+  {:height       2
+   :flex         1
+   :border-width 1
+   :border-color (get-color index position)
+   :margin-right (if end? 0 8)})
+
+(def carousel
+  {:height             92
+   ;; (padding-top) This insets issue needs a consistent implementation across all screens.
+   :padding-top        44
+   :position           :absolute
+   :top                0
+   :bottom             0
+   :left               0
+   :right              0
+   :z-index            2
+   :background-color   (colors/alpha "#000716" 1)
```

Comment:
Please can you move this to colors file, and also please can you provide a link for this?

Reply:
https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=3068%3A332610&t=pNRAiBQSw

<img width="268" alt="Screenshot 2023-03-04 at 00 24 38" src="https://user-images.githubusercontent.com/22799766/222859784-6e11fcbc-312c-4f8e-a48a-7baa9d6b6d28.png">
eWDLO7W-0

I will double check with design as what we can call this black as perhaps they need to adjust the design file colorset as well.